apps/mobile/src/components/headers/developer-console-header.tsx

```diff
@@ -0,0 +1,13 @@
+import { TitleHeader } from '@/components/headers/title-header';
```

Comment:
Maybe we can just get rid of this? 
![Screenshot 2025-03-28 at 14 52 48](https://github.com/user-attachments/assets/71004f48-4205-4822-9d5e-5c472cb201b1)

It's only for Developer tools so we don't really need it. Then it could help simplify how we handle the other Headers?

Reply:
Yeah, honestly I'd love to. Will do.